source/leuven/lecturer_s_procedure_to_request_student_accounts_ku_leuven_uhasselt.rst

```diff
@@ -10,43 +10,18 @@ several actions from the lecturer are required.
    `hpcinfo@.kuleuven.be <mailto:hpcinfo@kuleuven.be?subject=Accounts%20requests%20for%20students%20attending%20the%20course>`_
    providing a short description of the course and explanation why HPC
    facilities are necessary for teaching the course. Please also add the
-   attachment with the list of students attending the course (2 weeks
+   attachment with the list of students attending the course (1-2 weeks
    before the beginning of the course).
-#. Send the information to students that they have 1 week time window to
-   apply for the account (the last day when account creating can be
-   processed is the day before the course starts). Students should
-   follow the :ref:`generic procedure <generic access procedure>`, that
-   starts with
-   generating private-public key pair and ends with submitting the
-   public key via the `VSC account page`_. After 1 week the
-   lists of students that already submitted the request for the account
-   and corresponding vsc-account numbers will be send to the lecturer.
-#. The students should be informed to bring the private key with them to
-   be able to connect and attend the course.
+#. We advise to use :ref:`Open OnDemand <ood_t2_leuven>` service for the student to get access to the login nodes,
+   file browser and the job submission.
 #. Since introductory credits are supposed to be used for private
    projects (e.g. master thesis computations) we encourage to create the
    project which will be used for computations related to the course.
    This will also give a lecturer an opportunity of tracing the use of
    the cluster during the course. For more information about the
    procedure of creating the project please refer to the page
    :ref:`Slurm accounting <accounting_leuven>`.
-   Once the project is accepted, the students that already applied for
```

Comment:
I would replace it the text
 We encourage to create a specific 
   project, starting with "p_edu_", which will be used for computations related to the course.
   We will assign compute credits to this project. It also gives a lecturer an opportunity of tracing the use of
   the cluster during the course. After provisioning the student accounts, we will also add these to the given project.
For more information about the ...